qute.jdt/com.redhat.qute.jdt.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,25 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %pluginName
+Bundle-Vendor: %providerName
+Bundle-Localization: plugin
+Bundle-SymbolicName: com.redhat.qute.jdt.test;singleton:=true
+Bundle-Version: 0.11.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-11
+Require-Bundle: org.junit,
+ org.apache.commons.io,
+ org.apache.commons.lang3,
+ org.eclipse.jdt.ls.core;resolution:=optional,
+ com.google.guava,
+ org.eclipse.lsp4j,
+ org.eclipse.lsp4j.jsonrpc,
+ org.eclipse.core.runtime,
+ org.eclipse.jdt.core,
+ org.eclipse.jdt.core.manipulation,
+ org.eclipse.core.resources,
+ org.eclipse.ltk.core.refactoring,
+ org.eclipse.text,
+ com.redhat.qute.jdt,
+ org.eclipse.m2e.core

```

Comment:
`org.eclipse.m2e.lifecyclemapping.defaults` is missing, so it's not added to the TP. Without it, m2e doesn't bind project configurators (notably the JDT configurator) to maven plugin executions (maven-compiler-plugin). So the projects imported during tests don't get their classpath configured.